sick-fits/frontend/components/styles/NavStyles.js

```diff
@@ -50,10 +50,10 @@ const NavStyles = styled.ul`
       outline: none;
       &:after {
         width: calc(100% - 60px);
+        @media (max-width: 700px) {
+          width: calc(100% - 10px);
+        }
       }
```

Comment:
this was the main concern. when hovering over the nav items they would get janky since the media query was one curly bracket too low therefore causing the mq to be applied to the `&:before` psuedo element.